ipn/ipnlocal/local.go

```diff
@@ -3994,24 +3994,6 @@ func (b *LocalBackend) OperatorUserID() string {
 	return u.Uid
 }
 
-// TestOnlyPublicKeys returns the current machine and node public
-// keys. Used in tests only to facilitate automated node authorization
-// in the test harness.
-func (b *LocalBackend) TestOnlyPublicKeys() (machineKey key.MachinePublic, nodeKey key.NodePublic) {
```

Comment:
I believe this is used in corp test code: https://github.com/tailscale/corp/blob/41e9f6e1b55fd1836acea1036f4a9c4f2416dab8/ipn/e2e_test.go#L528-L532

Reply:
so it is 🤦🏼‍♂️